### PR TITLE
Fixes : #4273 Add link to user guide in About 

### PR DESCRIPTION
**Pull request by Rishavgupta12345**
Fixes: #4273  Add a link to the user guide in About 

screen short:
Screenshot 2022-03-07 193720
-

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/AboutActivity.java [hunk 1]
```diff
@@ -114,6 +116,10 @@ public void launchCredits(View view) {
         Utils.handleWebUrl(this, Uri.parse(Urls.CREDITS_URL));
     }
 
+    public void launchUserGuide(View view) {
```
Comment (madhurgupta10) [inline app/src/main/java/fr/free/nrw/commons/AboutActivity.java:119]: You can add test for this method

**Review by madhurgupta10 — CHANGES_REQUESTED** [review]

File: app/src/main/java/fr/free/nrw/commons/AboutActivity.java (hunk 1 above)
Comment (Rishavgupta12345) [reply app/src/main/java/fr/free/nrw/commons/AboutActivity.java:119]: done sir

**Review by Rishavgupta12345 — COMMENTED** [review]
Tests added

**Review by 4D17Y4 — APPROVED** [review]
LGTM

File: app/src/main/java/fr/free/nrw/commons/AboutActivity.java (hunk 1 above)
Comment (madhurgupta10) [reply app/src/main/java/fr/free/nrw/commons/AboutActivity.java:119]: Thanks for adding the Instrumentation test, you can also add a unit test for this method.

**Review by Rishavgupta12345 — COMMENTED** [review]
unit tests added @madhurgupta10  @4D17Y4

**Review by madhurgupta10 — APPROVED** [review]

**Comment by Rishavgupta12345** [comment]
as it is approved so please merge it

**Comment by 4D17Y4** [comment]
Thanks @Rishavgupta12345

File: app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt [hunk 2]
```diff
@@ -104,6 +104,15 @@ class AboutActivityTest {
                 IntentMatchers.hasData(Urls.CREDITS_URL)))
     }
 
+    @Test
+    @Ignore("Fix Failing Test")
```
Comment (madhurgupta10) [inline app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt:108]: @Rishavgupta12345 I just noticed that you are not actually running this test. Please remove the `@Ignore` annotation and fix the test.

Comment (4D17Y4) [reply app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt:108]: Most of the test in this file has the @Ignore annotation, I'm not sure if it is intended.

Comment (madhurgupta10) [reply app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt:108]: @4D17Y4  https://github.com/commons-app/apps-android-commons/pull/4832 should fix most of the tests

Comment (madhurgupta10) [reply app/src/androidTest/java/fr/free/nrw/commons/AboutActivityTest.kt:108]: I have also fixed the recent test added by @Rishavgupta12345

**Pull request merged**